registrations/tests/test_seatsreservation_put.py

```diff
@@ -26,110 +26,160 @@ def assert_update_seats_reservation(api_client, pk, reservation_data):
 
 
 @pytest.mark.django_db
-def test_update_seats_reservation(api_client, event, registration):
+def test_authenticated_admin_user_can_update_seats_reservation(
+    user_api_client, event, registration
+):
     registration.maximum_attendee_capacity = 2
-    registration.save()
+    registration.save(update_fields=["maximum_attendee_capacity"])
 
-    reservation = SeatReservationCode.objects.create(seats=1, registration=registration)
+    reservation = SeatReservationCodeFactory(seats=1, registration=registration)
     reservation_data = {
         "seats": 2,
         "registration": registration.id,
         "code": reservation.code,
     }
-    assert_update_seats_reservation(api_client, reservation.id, reservation_data)
+    assert_update_seats_reservation(user_api_client, reservation.id, reservation_data)
 
 
 @pytest.mark.django_db
-def test_seats_amount_has_not_limit_if_maximum_attendee_capacity_is_none(
+def test_authenticated_regular_user_can_update_seats_reservation(
+    user_api_client, event, registration, user
+):
+    user.get_default_organization().regular_users.add(user)
+    user.get_default_organization().admin_users.remove(user)
+
+    registration.maximum_attendee_capacity = 2
+    registration.save(update_fields=["maximum_attendee_capacity"])
+
+    reservation = SeatReservationCodeFactory(seats=1, registration=registration)
+    reservation_data = {
+        "seats": 2,
+        "registration": registration.id,
+        "code": reservation.code,
+    }
+    assert_update_seats_reservation(user_api_client, reservation.id, reservation_data)
+
+
+@pytest.mark.django_db
+def test_anonymous_user_cannot_update_seats_reservation(
     api_client, event, registration
+):
+    registration.maximum_attendee_capacity = 2
+    registration.save(update_fields=["maximum_attendee_capacity"])
+
+    reservation = SeatReservationCodeFactory(seats=1, registration=registration)
+    reservation_data = {
+        "seats": 2,
+        "registration": registration.id,
+        "code": reservation.code,
+    }
+    response = update_seats_reservation(api_client, reservation.id, reservation_data)
+    assert response.status_code == status.HTTP_401_UNAUTHORIZED
+
+
+@pytest.mark.django_db
+def test_seats_amount_has_not_limit_if_maximum_attendee_capacity_is_none(
+    user_api_client, event, registration
```

Comment:
`event` is not used... and applies to the rest of the file as well. What's up with these?

Reply:
Applies to `test_seatsreservation_post.py` as well.

Reply:
The unused event fixtures already existed and ended up in the new test cases as a result of using the older test cases as their basis. But now also these have been removed.